provider/cmd/pulumi-resource-purrl/schema.json

```diff
@@ -97,7 +97,7 @@
                     "type": "string"
                 },
                 "deleteResponseCodes": {
-                    "description": "The expected response code.",
+                    "description": "The expected response code(s) for deletion. Deprecated -- use `expectedDeleteResponseCodes` instead.",
```

Comment:
Hi @necaris,

can you use `deprecationMessage` instead?

see  https://www.pulumi.com/docs/guides/pulumi-packages/schema/#property